### ISSUE TITLE
Language "null" not supported by Atomic Calendar

### ISSUE THREAD

**Issue by dusan-ivanco** [totaldebug/atomic-calendar-revive#1782]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Everything is working, but I'm getting this warning in the console

Language "null" not supported by Atomic Calendar, request it https://github.com/totaldebug/atomic-calendar-revive/discussions/new?category=feature-requests

### Expected behaviour

No warning

### Mode

Event (default)

### Card Version

10.3.0

### Home Assistant Version

2026.6.0

### Configuration

```shell
- type: custom:atomic-calendar-revive
  entities:
    - entity: calendar.home
      maxDaysToShow: 7
  eventDateFormat: D ddd
  language: sk
```

### Relevant log output

```shell
atomic-calendar-revive.js?hacstag=2465497471030:38 Language "null" not supported by Atomic Calendar, request it https://github.com/totaldebug/atomic-calendar-revive/discussions/new?category=feature-requests
oo @ atomic-calendar-revive.js?hacstag=2465497471030:38
(anonymné) @ atomic-calendar-revive.js?hacstag=2465497471030:38
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by marksie1988** [comment]
Fixed in #1791 (commit aeda76f). Two problems addressed: (1) `localize()` now honours the card's `language` config option for UI strings (previously it only affected dayjs date formatting), and (2) empty / "null" / "undefined" language values (e.g. a stale `selectedLanguage` in localStorage, or calls made before `hass` is set) now fall back to English silently instead of logging the warning. Merged to `beta` and promoting to stable via #1793.
